ValetWatchTest Extension/InterfaceController.m

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
Precisely, the fact that Apple Watch doesn't have a unit test framework makes this quite difficult. We'll likely need to:
• Put `#import <XCTest/XCTest.h>` behind a `#if !TARGET_OS_WATCH`
• Manually re-define all of the `XCTAssert*` calls that are used within an `#if TARGET_OS_WATCH`
• Manually create a framework to call `setup`, `test_*` `teardown` on each `test_*` method that we have.

It would be great to use objc-runtime magic to find all the `test_*` methods on `ValetTests.m`.